mlrun/feature_store/api.py

```diff
@@ -840,6 +841,8 @@ def _ingest_with_spark(
             df = source.filter_df_start_end_time(df, timestamp_key)
         if featureset.spec.graph and featureset.spec.graph.steps:
             df = run_spark_graph(df, featureset, namespace, spark)
+        if isinstance(df, Response) and df.status_code == 400:
```

Comment:
why only 400?

Reply:
it is always 400 you can see it on server.run func , line 268 in server.py

Reply:
I changed it to be != 0